### DISCUSSION TITLE
COULD NOT SELECT WORKFLOWS

### DISCUSSION THREAD

**Discussion by hieultse03355** [codemagic-ci-cd/codemagic-docs#1260] [Q&A] [answered]
Hi guys, have a nice day
i got an issue about Starting new build on codemagic site. i was checking everything is fine in my codemagic.yaml but when i open codemagic site for my first build then i could not select any workflow and the button ' Start new build ' is grey and untouchable
Everything go fine for the last 2 weeks. But now i could not do any with the same setting in my codemagic.yaml:
![Screen Shot 2022-12-22 at 15 17 50](https://user-images.githubusercontent.com/14196282/209090122-60b3335f-3acb-4f06-96cf-b28153ba3385.png)
Could not pick workflows:
![Screen Shot 2022-12-22 at 15 17 59](https://user-images.githubusercontent.com/14196282/209090411-a5587fa4-c70d-4166-80a1-c6bbef10e2b2.png)
Thank you guys for answering

**Comment by icarusdust** [comment, marked as answer]
Hi @hieultse03355, I do not see any issues on our end. Perhaps refresh the page and try again., worth trying it with a different browser as well. Let me know how it goes.

**Comment by hieultse03355** [comment]
oke, maybe safari does something wrong here, it works on chorme as well, thank you so much
